docs/intro_07_bug_bounty.md

```diff
@@ -26,50 +26,59 @@ The scope of the bug bounty program includes the core contracts related to the f
 
 - *v1.1.1* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.1.1), [readme](https://github.com/gnosis/safe-contracts/blob/v1.1.1/README.md)) 
 - *v1.2.0* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.2.0), [readme](https://github.com/gnosis/safe-contracts/blob/v1.2.0/README.md)) 
+- *v1.3.0* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.3.0), [readme](https://github.com/gnosis/safe-contracts/blob/v1.3.0/README.md)) 
 
 The scope of the bug bounty also includes the [allowance module](https://github.com/gnosis/safe-modules/blob/47e2b486b0b31d97bab7648a3f76de9038c6e67b/allowances).
 
-#### In scope:
+### In scope:
```

Comment:
With these modified hierarchies, it still looks good and readable in the deployed version?

Reply:
Originally I did it the former way since I though it's better for overview. Especially further down with the longer titles of the found vulnerabilities. 
If it's looks okay, no objections to adapt though from my side.

Reply:
I hate this docusaurus xD

Reply:
![image](https://user-images.githubusercontent.com/2896048/117161367-228f2800-adc2-11eb-9760-f317e3ae13f2.png)